src/overlays/actors/ovl_En_Kusa/z_en_kusa.h

```diff
@@ -7,12 +7,33 @@ struct EnKusa;
 
 typedef void (*EnKusaActionFunc)(struct EnKusa*, GlobalContext*);
 
+typedef enum {
+    /* 0 */ ENKUSA_TYPE_BUSH,
+    /* 1 */ ENKUSA_TYPE_REGROW_GRASS,
```

Comment:
```suggestion
    /* 1 */ ENKUSA_TYPE_REGROWING_GRASS,
```
I know it's longer, but the ungrammatical version's bothering me more.